dll/win32/comdlg32/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 
 add_definitions(
     -D__WINESRC__
-    -D_WINE)
-
+    -D_WINE
+    -Dwcsdup=_wcsdup)
+remove_definitions(-D__ROS_LONG64__)
```

Comment:
Just a question for understanding: What motivation made you add this line?

Reply:
-D__ROS_LONG64__ is for https://gitlab.winehq.org/wine/wine/-/commit/05fef74867799ff57dcd1c7f248c37ac804ddaf5

so that stuff like DWORD compiles as LONG UINT instead of UINT

that -Dwcsdup is to fix:

`D:\reactos\output-MinGW-i386/../dll/win32/comdlg32/printdlg.c:2774: undefined reference to `_imp__wcsdup'`